Makefile.lambda

```diff
@@ -1,9 +1,14 @@
 # public.ecr.aws/l1s5s8m2 is the public ecr repository for integration testing
-AWS_ACCOUNT := "442632209887"
-LAMBDA_ECR_REPO := "442632209887.dkr.ecr.us-west-2.amazonaws.com"
-LAMBDA_CONTAINER_REPO_NAME := "iambic_private_ecr"
-LAMBDA_IMAGE_LABEL := "latest"
-PROFILE_NAME := "iambic_test_org_spoke_account_1/iambic_test_org_spoke_account_1_admin"
+# Override variables below by setting environment variables and running. Example:
+# AWS_ACCOUNT=242350334841 AWS_REGION=us-east-1 LAMBDA_CONTAINER_REPO_NAME=iambic-ecr-public/iambic/iambic PROFILE_NAME=test/test_admin make -f Makefile.lambda auth_to_ecr upload_docker_lambda
```

Comment:
242350334841 is not an account in iambic test org. so in case you don't want to introduce this to the repo.